common/reals/real_thandle_helper.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifndef REAL_THANDLE_HELPER_H
+#define REAL_THANDLE_HELPER_H
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_pal/thandle.h"
+#include "c_pal/thandle_ll.h"
+
+#include "c_pal/gballoc_hl.h"
+#include "real_gballoc_hl.h"
+
+// Creates a THANDLE alias of TYPE called REAL_TYPE
+// Must have ENABLE_MOCKS undefined
+// Recommend including real_interlocked_renames.h before this call and real_interlocked_undo_renames.h after this call
```

Comment:
>ecommend including real_interlocked_renames.h before this call and real_interlocked_undo_renames.h after this call [](http://example.com/codeflow?start=4&length=114)

I think we should have this in the MD as a note and as part of the example code there